### ISSUE TITLE
Behavior of reactions made by admin in group is utterly random

### ISSUE THREAD

**Issue by php4fan** [telegramdesktop/tdesktop#27419]
### Steps to reproduce

1. Have a supergroup where you are an admin with permission to stay anonymous
2. there's a bot which is also admin who posts in the group
3. (You) react to some of the bot's posts with an emoji

### Expected behaviour

Something consistent.

If the fact that I am anonymous means I cannot react, then I shouldn't have the UI to react in the first place.

If being anonymous means my reactions are anonymous, then they should always work consistently, that is, show up and remain, be displayed without my avatar (if this is how they're going to be seen by others, it would be stupid to show them with the avatar to me), and be seen across devices and be sent to the bot via the API (yes, the bot is subscribed to `message_reaction` and `message_reaction_count` type updates)

If being anonymous means I cannot react, and for whatever reason you can't be smart enough to avoid showing a UI to do something that I can't do, then the reaction should disappear immediately, and with some feedback that let me understand what happened, e.g. an error message saying I cannot react and exactly why.

If reactions given by an anonymous admin are not supposed to be anonymous (which would be stupid) then they should work consistently, as non-anonymous reactions.

### Actual behaviour

When I (an anonymous admin) react to the bot posts in Telegram Desktop, RANDOMLY any of these happen:

- the reaction is displayed (to me in TD), with my avatar next to it (which makes no sense given that I am anonymous)
- - and then it disappears a few seconds later OR NOT
OR
- the reaction is displayed without my avatar next to it (as expected)
- - and then it disappears a few seconds OR NOT

In either case, the reaction never shows up on the mobile app, and the update is never sent to the bot (which is receiving updates for reactions by other users, both other anonymous admins, non-anonymous admins, and regular users). 

Basically it seems the reaction never leaves Telegram Desktop, and the way it behaves locally is utterly random and inconsistent (sometimes it disappears sometimes not; sometimes it has my avatar sometimes not). And no error message is ever displayed indicating any fault of any sort.

### Operating system

Manjaro Linux

### Version of Telegram Desktop

4.14.9

### Installation source

Flatpak

**Comment by php4fan** [comment]
Again, you should suppress that stupid bot.

**Comment by Aokromes** [comment]
> Again, you should suppress that stupid bot.

again, stupid bot helps us to find already fixed issues missed to close.

**Comment by php4fan** [comment]
> again, stupid bot helps us to find already fixed issues missed to close.

LOL seriously?

If that's really what you're using it for, you don't seem to be doing anything to prevent it from closing issues on false positives.

All the issues reported by me that the bot marked as stalled (usually because there was no reply from your side, and I don't think I've ever seen one that had been fixed), one of two things happened: either I replied myself to prevent them from being closed, or they got closed.


**This one still exists**, in case you doubt.

**Comment by Aokromes** [comment]
> > again, stupid bot helps us to find already fixed issues missed to close.
> 
> LOL seriously?
> 
> If that's really what you're using it for, you don't seem to be doing anything to prevent it from closing issues on false positives.
> 
> All the issues reported by me that the bot marked as stalled (usually because there was no reply from your side, and I don't think I've ever seen one that had been fixed), one of two things happened: either I replied myself to prevent them from being closed, or they got closed.
> 
> **This one still exists**, in case you doubt.

we give fairly decent timeout to close issue after slate (30 days), and again decent time (45 days) after close issue to lock, that's 75 days or 2 months and half after warning.

**Comment by foadk** [comment]
> Don't worry though; if this is an error, let us know with a comment and we'll be happy to reopen the issue.

Yes, this bug still exists and is super annoying. I'm an administrator of a group consisting of just me and the owner. We are both anonymous. I'm seeing their reactions as group name.
I'm able to select and send reactions to messages, but they're not seeing any of them.
Is there any plans to fix?

**Comment by php4fan** [comment]
Why are you ignoring this??

This still happens systematically in a few specific groups with specific (admin) users, and has been going on consistently for months. These reactions are not being sent to Telegram, or at least not to bots that are listening to updates in the chat. This has disastrous consequences. We have bots that do automated moderation tasks based on reactions by the admins, but when this happens (which is always for the affected users in the affected chats), the reactions are never received by the bot via the API. All other reactions are received as expected.

**Comment by 23rd** [comment]
Should be better now.

**Comment by php4fan** [comment]
> Should be better now.

With what version?

**Comment by 23rd** [comment]
Since v5.11.0.

**Comment by php4fan** [comment]
Oh, then nope, it's still exactly the same in 5.1.11.

**Comment by php4fan** [comment]
The stupid bot again.
This issue still exists. Please reopen.

**Comment by php4fan** [comment]
Still observing this issue systematically with some users in some groups.

**Comment by php4fan** [comment]
Still observing this issue systematically with some users in some groups.
